frontend/packages/dev-console/src/components/edit-application/EditApplication.tsx

```diff
@@ -72,7 +72,8 @@ const EditApplication: React.FC<EditApplicationProps> = ({
   };
 
   const handleSubmit = (values, actions) => {
-    const imageStream = builderImages && builderImages[values.image.selected].obj;
+    const imageStream =
+      values.image.selected && builderImages ? builderImages[values.image.selected].obj : null;
```

Comment:
could this be? then you wouldn't need to check individually
const imageStream =  builderImages ? _.get( builderImages, [values, image, selected, obj ], null ) : null;

Reply:
@abhinandan13jan did you try running this? This usage is wrong since there is no such properties such as `values`, `image` or `selected` in `builderImages`. If you see `values.image.selected` gives me the name of the selected image which is a property of `builderImages` and I want the obj property of that image.

Reply:
This might be helpful https://lodash.com/docs/4.17.15#get.

Reply:
My bad.